manifest.json

```
@@ -17,26 +12,29 @@
         "https://trello.com/*"
       ],
       "js": [
-        "vendor/browser-polyfill.js",
-        "content_script.js"
+        "scripts/content_script.js",
+        "scripts/trektor.js"
```

Hier die Reihenfolge noch aufräumen?

```suggestion
        "scripts/trektor.js",
        "scripts/content_script.js"
```